scripts/init_app_dir.sh

```diff
@@ -9,7 +9,10 @@ mkdir -p \
   "${APP_DIR}/usr/lib" \
   "${APP_DIR}/usr/qml" \
   "${APP_DIR}/usr/i18n" \
-  "${APP_DIR}/usr/plugins/platforminputcontexts"
+  "${APP_DIR}/usr/plugins/platforminputcontexts" \
+  "${APP_DIR}/etc/reader.conf.d" \
+  "${APP_DIR}/usr/lib/pcsc/drivers" \
+  "${APP_DIR}/usr/sbin"
```

Comment:
Why not `bin`? I dont think `sbin` makes sense in appimage.

Reply:
I have no good reason for this, I probably chose `sbin` because i found `psc` libs at `usr/local/sbin/pcscd`
I can experiment by keeping them in `bin` and if everything works well I don't mind using `bin` instead of `sbin`

Reply:
works with `bin` as well. Thanks for the suggestion.